### PR TITLE
Update Simplified Chinese transaction

### PR DESCRIPTION
**Pull request by UraraChiya**

### CODE REVIEW

File: src/main/resources/assets/functionalstorage/lang/zh_cn.json [hunk 1]
```diff
@@ -1,82 +1,97 @@
...
-  "linkingtool.use": "* 对空气潜行+右击以变更模式或清空频率。 \n* 左击末影抽屉记录其频率。 \n* 对空气右击以变更连接操作。 \n\n右击控制器以设置工具，然后连接附近的抽屉。 \n\n手持该工具时，将会高亮所选控制器以及与其连接的抽屉。",
-  "storageupgrade.desc": "存储上限×",
-  "upgrade.type": "种类：",
+  "linkingtool.linkingmode": "链接模式：",
+  "linkingtool.linkingmode.multiple": "多个",
+  "linkingtool.linkingmode.multiple.desc": "在 2 点之间链接多个抽屉",
+  "linkingtool.linkingmode.single": "单个",
+  "linkingtool.linkingmode.single.desc": "将抽屉链接到控制器",
+  "linkingtool.use": "* 潜行 + 右键空气单击可更改模式或清除频率。\n* 左键单击末影抽屉以存储其频率。\n* 右键空气单击可以更改动作。\n\n右键单击控制器以设置该工具，然后使用它附近的抽屉进行链接。\n\n按持有该工具时，它将显示连接到所选控制器的抽屉。",
+  "storageupgrade.desc.fluid": "将方块液体存储量乘以 ",
```
Comment (ZHAY10086) [inline src/main/resources/assets/functionalstorage/lang/zh_cn.json:92]:
```suggestion
  "storageupgrade.desc.fluid": "流体存储上限× ",
```

File: src/main/resources/assets/functionalstorage/lang/zh_cn.json [hunk 2]
```diff
@@ -1,82 +1,97 @@
...
+  "storageupgrade.desc.item": "将方块物品存储量乘以 ",
```
Comment (ZHAY10086) [inline src/main/resources/assets/functionalstorage/lang/zh_cn.json:93]:
```suggestion
  "storageupgrade.desc.item": "物品存储上限×",
```

File: src/main/resources/assets/functionalstorage/lang/zh_cn.json [hunk 3]
```diff
@@ -1,82 +1,97 @@
...
+  "block.functionalstorage.cherry_4": "樱花抽屉（2x2）",
   "block.functionalstorage.compacting_drawer": "压缩抽屉",
   "block.functionalstorage.compacting_framed_drawer": "镶框压缩抽屉",
-  "block.functionalstorage.crimson_1": "绯红木抽屉（1x1）",
-  "block.functionalstorage.crimson_2": "绯红木抽屉（1x2）",
-  "block.functionalstorage.crimson_4": "绯红木抽屉（2x2）",
-  "block.functionalstorage.dark_oak_1": "深色橡木抽屉（1x1）",
-  "block.functionalstorage.dark_oak_2": "深色橡木抽屉（1x2）",
-  "block.functionalstorage.dark_oak_4": "深色橡木抽屉（2x2）",
+  "block.functionalstorage.controller_extension": "控制器扩展",
```
Comment (ZHAY10086) [inline src/main/resources/assets/functionalstorage/lang/zh_cn.json:14]:
```suggestion
  "block.functionalstorage.controller_extension": "控制扩展器",
```
可能更顺口一点

File: src/main/resources/assets/functionalstorage/lang/zh_cn.json [hunk 4]
```diff
@@ -1,82 +1,97 @@
...
+  "block.functionalstorage.fluid_1": "流体抽屉（1x1）",
+  "block.functionalstorage.fluid_2": "流体抽屉（1x2）",
+  "block.functionalstorage.fluid_4": "流体抽屉（2x2）",
   "block.functionalstorage.framed_1": "镶框抽屉（1x1）",
   "block.functionalstorage.framed_2": "镶框抽屉（1x2）",
   "block.functionalstorage.framed_4": "镶框抽屉（2x2）",
-  "block.functionalstorage.jungle_1": "丛林木抽屉（1x1）",
-  "block.functionalstorage.jungle_2": "丛林木抽屉（1x2）",
-  "block.functionalstorage.jungle_4": "丛林木抽屉（2x2）",
+  "block.functionalstorage.framed_controller_extension": "镶框控制器扩展",
```
Comment (ZHAY10086) [inline src/main/resources/assets/functionalstorage/lang/zh_cn.json:28]:
```suggestion
  "block.functionalstorage.framed_controller_extension": "镶框控制扩展器",
```
同上

**Comment by UraraChiya** [comment]
@ZHAY10086 现在应该已修正大部分错误

**Comment by ZHAY10086** [comment]
辛苦你了

**Comment by Buuz135** [comment]
Is this fine to merge?

**Comment by ZHAY10086** [comment]
I think it's ready.
@UraraChiya

**Pull request merged**